docs/sources/tempo/metrics-generator/service_graphs.md

```diff
@@ -43,6 +44,18 @@ Each emitted metrics series have the `client` and `server` label corresponding w
   tempo_service_graph_request_total{client="app", server="db", connection_type="database"} 20
 ```
 
+### Virtual nodes
+
+Virtual nodes are nodes that form part of the lifecycle of a trace,
```

Comment:
@mapno Should I remove this section from the backport? It's in the changelog as unreleased (#2365).